### ISSUE TITLE
gopass show can't copy to clipboard from WSL

### ISSUE THREAD

**Issue by slokhorst** [gopasspw/gopass#1664]
I'm using gopass in an WSL 2 instance (Windows Subsystem for Linux). This works great with one exception: every time I run `gopass generate test` it says:
```
Error: failed to copy to clipboard: failed to write to clipboard: exit status 1
```

I found that it is possible to copy text to the clipboard with `echo 'test' | clip.exe`, but I'd understand if this is too niche to implement in gopass.

As an alternative, maybe would be a sane default to just print the generated password if copying to the clipboard fails?

Windows 10 20H2 with WSL2
Ubuntu 20.04.1
gopass 1.9.0

**Comment by AnomalRoil** [comment]
Did you try installing `xclip` on your WSL? Does it work with it?

We could give it a try. Since WSL doesn't support hardware PGP tokens, it's
low priority for now, but this might happen next year.

**Comment by freijon** [comment]
Try using `wclip`. It's an exe which you can put somewhere in Windows and make a symbolic link in WSL like `ln -s <path-to-exe> /usr/local/bin/wclip. Just pipe the output of gopass to wclip.

**Comment by AnomalRoil** [comment]
The easiest way for us is to wait until support for WSL is added to `atotto/clipboard` ^

**Comment by dominikschulz** [comment]
Yes, I think we need to wait for atotto/clipboard#45 to be resolved.

**Comment by Hazegard** [comment]
Hi!

As a workaround I have this little script called `xclip` in my PATH:

```bash
#!/bin/bash
cat | clip.exe
```

It works enough to have passwords and otp codes in my windows clipboard.

**Comment by AnomalRoil** [comment]
That's a nice workaround, indeed. 😄

**Comment by slokhorst** [comment]
@Hazegard that works great, thanks!

**Comment by AnomalRoil** [comment]
We might consider switching to https://github.com/golang-design/clipboard which is reported to work with WSL on windows.

**Comment by dominikschulz** [comment]
No, that's using CGo. We can not introduce CGo dependencies with our current build system.
